LiskHQ/lisk
# Node inspect can't be run with default child process - Closes #3193

### What was the problem?

After the introduction of child processes it was not possible anymore to run nodejs' inspector

### How did I fix it?

By adding code to assign different random ports to the debugger if the environmental variable NODE_DEBUG is set to true

### How to test it?

Run `NODE_DEBUG=true node --inspect src/index.js` and the debugger and inspector should work

### Review checklist

* The PR resolves #3193
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
